packages/web-react/src/components/TextField/README.md

```diff
@@ -0,0 +1,26 @@
+# TextField
+
+TextField enables the user to type in text information. It has input, label,
+and optional message. It could be disabled or have an error state. The label could be hidden
+and show if the input is required.
+
+```jsx
+<TextField id="example" name="example" required hasError messsage="validation failed" />
+```
+
+## Available props
+
+| Name            | Type    | Description                    |
+| --------------- | ------- | ------------------------------ |
+| `id`            | string  | Input and label identification |
+| `name`          | string  | Input name                     |
+| `label`         | string  | Label text                     |
+| `placeholder`   | string  | Input placeholder              |
+| `value`         | string  | Input value                    |
+| `message`       | string  | Validation or help message     |
+| `disabled`      | boolean | Whether is field disabled      |
+| `required`      | boolean | Whether is field required      |
+| `hasError`      | boolean | Whether has field an error     |
```

Comment:
There will be two more validation states in `PasswordField` and I think it could be convenient to have the same syntax of validation state everywhere. Maybe `validationState="error"`? More states can be added later without making it a BC.

![obrazek](https://user-images.githubusercontent.com/5614085/155553006-010d1fbc-e870-4e5e-aec5-3986cb902af4.png)

Reply:
Ok, changed.